.github/buildomat/jobs/netstack-prepare.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# This script is used to prepare the tools used in the
+# netstack validation environment
+#:
+#: name = "netstack-prepare"
+#: variety = "basic"
+#: target = "helios"
+#: rust_toolchain = "nightly"
+#:
+#: output_rules = [
+#:   "=/out/build-tools.tar.xz",
+#: ]
+#:
+#: access_repos = [
+#:   "oxidecomputer/falcon-image-netstack",
+#:   "oxidecomputer/falcon",
+#:   "oxidecomputer/propolis",
+#:   "oxidecomputer/testbed",
+#: ]
+#:
+#: [[publish]]
+#: series = "tools"
+#: name = "build-tools.tar.xz"
+#: from_output = "/out/build-tools.tar.xz"
+#:
+#: [dependencies.build]
+#: job = "netstack-image"
+#
+
+set -e
+set -o pipefail
+set -o xtrace
+
+export RUST_LOG=debug
+
+# List the host's interfaces
+dladm show-link
+topdir=$(pwd)
+
+mkdir bin
+
+pushd bin
+curl -OL https://github.com/stedolan/jq/releases/download/jq-1.4/jq-solaris11-64
```

Comment:
is there something wrong with the `jq` from `pkg`?

Reply:
`jq` doesn't exist on the baremetal ci server (it's _very_ barebones), so we have to bring our own. I think I tried copying the one that was installed but it was dynamically linked to a library that the baremetal server didn't have.